### ISSUE TITLE
How to Achieve YOLOv6n Accuracy as Stated in Ultralytics Documentation?

### ISSUE THREAD

**Issue by tino926** [ultralytics/ultralytics#17600]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I am experimenting with Ultralytics and training the YOLOv6n model. How can I achieve the same accuracy as mentioned in <https://docs.ultralytics.com/models/yolov6/>? The command provided on that page seems to be an example for the very small COCO8 dataset. What parameters should I use, such as the number of epochs, to replicate the stated accuracy on full coco dataset?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @tino926, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com) where you'll find a wealth of information to help you get started with YOLOv6n, including guidance on achieving optimal model performance.

If you're aiming to replicate the results from our documentation, it's crucial to follow the training protocols closely, including dataset preparation and hyperparameters. If this is a custom training ❓ Question, please provide additional information, including dataset examples and training logs. Also, ensure you're following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join our Ultralytics community for discussions and support. For real-time chat, head over to [Discord](https://ultralytics.com/discord) 🎧. Prefer more in-depth conversations? Check out our [Discourse](https://community.ultralytics.com). Alternatively, engage with fellow users on our [Subreddit](https://reddit.com/r/ultralytics).

## Upgrade

Make sure you are using the latest `ultralytics` package to avoid running into any fixed issues. Upgrade in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

You can run YOLO in any of the following verified environments, each preconfigured with dependencies like [CUDA](https://developer.nvidia.com/cuda), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

When this badge is green, it indicates all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are passing. This helps assure the reliability of YOLO across macOS, Windows, and Ubuntu environments.

Please note that this is an automated response. An Ultralytics engineer will also review your question and provide additional assistance soon.

**Comment by Y-T-G** [comment]
You need to train on the COCO dataset (`data="coco.yaml"`).

You can follow the training configuration given in the original repo.

https://github.com/meituan/YOLOv6/blob/main/configs/yolov6n.py
